platform/linux-generic/odp_packet.c

```diff
@@ -2879,3 +2879,44 @@ int odp_packet_payload_offset_set(odp_packet_t pkt, uint32_t offset)
 
 	return 0;
 }
+
+int odp_packet_tx_compl_request(odp_packet_t pkt, const odp_packet_tx_compl_opt_t *opt)
+{
+	(void)pkt;
+	(void)opt;
+
+	return -1;
+}
+
+int odp_packet_has_tx_compl_request(odp_packet_t pkt)
+{
+	(void)pkt;
+
+	return -1;
```

Comment:
This needs to return zero. We don't support it, so packet cannot have a request.